src/blog/controllers/BlogCtrl.php

```diff
@@ -7,7 +7,7 @@
 /**
 * The blog controller, get calls for display of posts
 */
-class Blog extends Controller
+class BlogCtrl extends Controller
```

Comment:
I liked Blog class better 😒, BlogController would be nice too

Reply:
Doesnt matter, controller classes shall not be used from codebase from now on